### PR TITLE
backend: add ManualReconnect method and call on iOS

### PR DESCRIPTION
**Pull request by benma**
When the iOS app goes into the background, its TCP connections (Electrum server connetions) are killed after a while (~30s).

When the user goes back into the app, we want to immediatelly trigger a reconnect, not waiting for the regular 30s reconnect timer.

Depends on https://github.com/BitBoxSwiss/block-client-go/pull/6 (when merged I'll update the local dep here)

### CODE REVIEW

File: backend/bridgecommon/bridgecommon.go [hunk 1]
```diff
@@ -166,6 +166,18 @@ func UsingMobileDataChanged() {
 	})
 }
 
+// ManualReconnect exposes the ManualReconnect backend method.
+func ManualReconnect() {
+	mu.Lock()
```
[the remark this answers is not in the document]
Comment (Beerosagos) [reply backend/bridgecommon/bridgecommon.go:171]: And also in some other functions in this file, I think.

File: frontends/ios/BitBoxApp/BitBoxApp/BitBoxAppApp.swift [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import SwiftUI
 import Mobileserver
 import LocalAuthentication
+import UIKit
```
Comment (Beerosagos) [inline frontends/ios/BitBoxApp/BitBoxApp/BitBoxAppApp.swift:11]: What is this used for?

File: backend/bridgecommon/bridgecommon.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (benma) [reply backend/bridgecommon/bridgecommon.go]: I did for this one, will make a PR for the others :+1:

File: frontends/ios/BitBoxApp/BitBoxApp/BitBoxAppApp.swift (hunk 2 above)
Comment (benma) [reply frontends/ios/BitBoxApp/BitBoxApp/BitBoxAppApp.swift]: This line somehow jumped from https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2939 to here :sweat_smile: 

Removed

**Review by Beerosagos — APPROVED** [review]
utACK

**Pull request merged**